core/src/main/java/de/danoeh/antennapod/core/feed/LocalFeedUpdater.java

```diff
@@ -122,6 +111,40 @@ private static void tryUpdateFeed(Feed feed, Context context) throws IOException
         DBTasks.updateFeed(context, feed, removeUnlistedItems);
     }
 
+    /**
+     * Returns the image URL for the local feed.
+     */
+    @NonNull
+    static String getImageUrl(@NonNull Context context, @NonNull DocumentFile documentFolder) {
+        String imageUrl = null;
+
+        // look for special file names
+        for (String iconLocation : PREFERRED_FEED_IMAGE_FILENAMES) {
+            DocumentFile image = documentFolder.findFile(iconLocation);
+            if (image != null) {
+                imageUrl = image.getUri().toString();
+                break;
```

Comment:
Are there cases where the file is not `null` but its uri is? If there are no such cases, I think the code would be easier to read if you just used `return image.getUri().toString()` here.

Reply:
No, there was no special reason for the code style. You are right, I switched the code to early return and it is more readable now.